contracts/evmscript/executors/CallsScript.sol

```diff
@@ -3,10 +3,10 @@ pragma solidity 0.4.18;
 // Inspired by https://github.com/reverendus/tx-manager
 
 import "../ScriptHelpers.sol";
-import "../IEVMScriptExecutor.sol";
+import "./BaseEVMScriptExecutor.sol";
 
 
-contract CallsScript is IEVMScriptExecutor {
+contract CallsScript is BaseEVMScriptExecutor {
     using ScriptHelpers for bytes;
```

Comment:
Why are we removing the interface here? I think we should still enforce that `CallScript` complies with `IEVMScriptExecutor` interface.

Reply:
Right, `BaseEVMScriptExecutor` should be `IEVMScriptExecutor`